packages/fuel-gauge/src/std-lib-string.test.ts

```diff
@@ -0,0 +1,91 @@
+import { generateTestWallet } from '@fuel-ts/wallet/test-utils';
+import {
+  type Contract,
+  bn,
+  Predicate,
+  Wallet,
+  Address,
+  BaseAssetId,
+  Provider,
+  FUEL_NETWORK_URL,
+} from 'fuels';
+
+import predicateStdString from '../fixtures/forc-projects/predicate-std-lib-string';
+import predicateStdStringAbi from '../fixtures/forc-projects/predicate-std-lib-string/out/debug/predicate-std-lib-string-abi.json';
+
+import { getScript, getSetupContract } from './utils';
+
+const setupContract = getSetupContract('std-lib-string');
+let contractInstance: Contract;
+beforeAll(async () => {
+  contractInstance = await setupContract();
+});
+
+const setup = async (balance = 5_000) => {
+  const provider = await Provider.create(FUEL_NETWORK_URL);
+
+  // Create wallet
+  const wallet = await generateTestWallet(provider, [[balance, BaseAssetId]]);
+
+  return wallet;
+};
+
+describe('std-lib-string Tests', () => {
+  it('should test std-lib-string return', async () => {
+    const { value } = await contractInstance.functions.return_dynamic_string().call<string>();
+    expect(value).toBe('Hello World');
+  });
+
+  it('should test std-lib-string input', async () => {
+    const INPUT = 'Hello World';
+
+    await contractInstance.functions.accepts_dynamic_string(INPUT).call<number[]>();
+
+    expect(true).toBe(true);
+  });
+
+  it('should test bytes input [predicate-std-lib-string]', async () => {
```

Comment:
Should this be `String` input rather than `bytes`?

```suggestion
  it('should test String input [predicate-std-lib-string]', async () => {
```